### PR TITLE
Separated load magics from start session

### PR DESCRIPTION
**Pull request by aggFTW**
This will make it easier to add configs

### CODE REVIEW

File: remotespark/sparkkernelbase.py [hunk 1]
```diff
@@ -9,35 +9,40 @@
...
+
         self.logger = Log(self.client_name)
-        self.already_ran_once = False
+        self.session_started = False
         self._fatal_error = None
 
         # Disable warnings for test env in HDI
         requests.packages.urllib3.disable_warnings()
 
+        if "testing" not in kwargs.keys():
```
Comment (msftristew) [inline remotespark/sparkkernelbase.py]: Try `if not kwargs.get("testing", False)`.  This will work even if I pass in `testing=False` as a keyword argument

**Comment by msftristew** [comment]
I'm confused about the nature of the change you made.  In what way did you "separate load magics from start session"?  Can you point me to the block of code which does this?

**Comment by aggFTW** [comment]
_start_session and _initialize_magics used to be the same.

File: remotespark/sparkkernelbase.py (hunk 1 above)
Comment (aggFTW) [inline remotespark/sparkkernelbase.py]: Already made the change but will add in next PR.

**Comment by msftristew** [comment]
I see.  Looks good, I sign off

**Pull request merged**